portal/templates/profile_macros.html

```diff
@@ -1687,7 +1688,21 @@ <h4 id="communicationsLoc" class="profile-item-title index-item">{{ _("Communica
                 </div>
             </div>
         </div>
-    {%- endif %}
+    {%- elif person and person.has_role(ROLE.STAFF) and current_user and current_user.has_role(ROLE.STAFF_ADMIN)-%}
```

Comment:
Is the check for `ROLE.STAFF` redundant with the check for `ROLE.STAFF_ADMIN`? ie are all `STAFF_ADMIN` also `STAFF`? Not entirely familiar with story details so disregard if I'm missing something.